docs/General/Manuals/client-manual.rst

```diff
@@ -1269,6 +1269,50 @@ left-click of the icon will bring up the :guilabel:`Nations View` as shown in Fi
 Research View
 ^^^^^^^^^^^^^
 
+The :guilabel:`Research View` is a separate page showing the technology research tree. This is the page
+where you instruct your scientists to research technologies. When you click the icon for it as shown
```

Comment:
"...your scientists ~~to research technologies~~ *which technologies to research*."

"button" instead of "icon"